build.gradle

```diff
@@ -121,12 +122,15 @@ uploadArchives {
 			beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
 			
 			// upload to sonatype OSS (snapshot/release)
+			/*
 			repository(url: this.version.endsWith('-SNAPSHOT') ?
 					'https://oss.sonatype.org/content/repositories/snapshots' :
 					'https://oss.sonatype.org/service/local/staging/deploy/maven2') {
 				authentication(userName: this.hasProperty('sonatypeUsername') ? sonatypeUsername : '',
 					password: this.hasProperty('sonatypePassword') ? sonatypePassword : '')
 			}
+			*/
+			repository(url: mavenLocal().url)
```

Comment:
Gah!  Sorry, sloppy work on my part.  Fixed now.  Thanks for `gradlew install` trick :)

I'm used to `maven-publish` plugin, which adds a `publishToMavenLocal` task.